dbld/images/devshell.dockerfile

```diff
@@ -8,3 +8,4 @@ RUN /dbld/builddeps install_perf
 RUN /dbld/builddeps install_apt_packages
 RUN /dbld/builddeps install_pip2
 RUN /dbld/builddeps install_pip_packages
+RUN /dbld/builddeps install_bison_from_source
```

Comment:
I missed this one: why do we need this?
We inherit bison from `balabit/syslog-ng-tarball` image:
https://github.com/syslog-ng/syslog-ng/blob/master/dbld/images/tarball.dockerfile#L1-L10

```
$ docker run -it --rm balabit/syslog-ng-devshell bash
root@c1c04bd96a0b:/# 
root@c1c04bd96a0b:/# 
root@c1c04bd96a0b:/# bison --version
bison (GNU Bison) 3.7.6
Written by Robert Corbett and Richard Stallman.

Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

```

Reply:
We don't need this. It would have been its own commit explaining why we need this if it was on purpose. I'll remove.